packages/react/src/Pagination/Pagination.tsx

```diff
@@ -10,6 +10,8 @@ import type { ForwardedRef, HTMLAttributes } from 'react'
 import { Icon } from '../Icon/Icon'
 
 export type PaginationProps = {
```

Comment:
The changes in this PR warrant a rethink of the Pagination api I think. Terms like `ariaLabel` aren't correct anymore. We discussed this in #ads-frontend, what do you think of the following api + descriptions?

```
export type PaginationProps = {
  /** The accessible name for the Pagination component. */
  visuallyHiddenLabel?: string
  /** The maximum amount of pages shown. Minimum value: 5. */
  maxVisiblePages?: number
  /** The accessible name for next page-button. */
  nextVisuallyHiddenLabel?: string
  /** The visible label for the next page-button. */
  nextLabel?: string
  /** A function to run when the page number changes. */
  // eslint-disable-next-line no-unused-vars
  onPageChange?: (page: number) => void
  /** The current page number. */
  page?: number
  /** The accessible name for the previous page-button. */
  previousVisuallyHiddenLabel?: string
  /** The visible label for the previous page-button. */
  previousLabel?: string
  /** The total amount of pages. */
  totalPages: number
} & HTMLAttributes<HTMLElement>
```

Reply:
Yeah, I think we should change the API. Can it be in-scope for this PR though? Maybe better to do it right away?

Reply:
Changed